requirements.txt

```diff
@@ -5,7 +5,7 @@ sympy>=1.5
 tensorflow>=2.0
 tensorboard>=2.0
 networkx>=2.0
-quantum-blackbird==0.2.3
+git+git://github.com/XanaduAI/blackbird@master#egg=blackbird
```

Comment:
This is causing the readthedocs to fail, and since a new release of Blackbird is around the corner I'll probably remove this and pin it to the latest release instead.